pypeit/par/pypeitpar.py

```diff
@@ -614,6 +614,11 @@ def __init__(self, method=None, pixelflat_file=None, spec_samp_fine=None,
                                    'extracted from the slit; \'continue\' - ignore the ' \
                                    'flat-field correction, but continue with the reduction.'
 
+        defaults['ref_idx'] = 0
+        dtypes['ref_idx'] = int
+        descr['ref_idx'] = 'The index of a reference slit (0-indexed) used for estimating ' \
```

Comment:
Please specify that this parameter is only needed if slit_illum_relative is True or being performed.

Reply:
Done!